app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -323,6 +326,70 @@ public static BgReading create(double raw_data, double filtered_data, Context co
                 }
 
                 bgReading.calculated_value = Math.min(400, Math.max(40, bgReading.calculated_value));
+                Log.w(TAG, "create: NEW VALUE CALCULATED AT: " + bgReading.calculated_value);
+
+                bgReading.save();
+                bgReading.perform_calculations();
+                Notifications.getInstance(context).notificationSetter(context);
+                BgSendQueue.addToQueue(bgReading, "create", context);
+            }
+        }
+        Log.w("BG GSON: ",bgReading.toS());
+
+        return bgReading;
+    }
+    public static BgReading create(double raw_data, Context context, Long timestamp) {
```

Comment:
Not sure if this came from my original support code or where. I only originally added the one that accepted raw and filtered. It may be a cut-paste error. 

 
John Stevens
M - +61 418526349

> On 20 May 2015, at 06:58, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java:
> 
> > @@ -323,6 +326,70 @@ public static BgReading create(double raw_data, double filtered_data, Context co
> >                  }
> >  
> >                  bgReading.calculated_value = Math.min(400, Math.max(40, bgReading.calculated_value));
> > +                Log.w(TAG, "create: NEW VALUE CALCULATED AT: " + bgReading.calculated_value);
> > +
> > +                bgReading.save();
> > +                bgReading.perform_calculations();
> > +                Notifications.getInstance(context).notificationSetter(context);
> > +                BgSendQueue.addToQueue(bgReading, "create", context);
> > +            }
> > +        }
> > +        Log.w("BG GSON: ",bgReading.toS());
> > +
> > +        return bgReading;
> > +    }
> > +    public static BgReading create(double raw_data, Context context, Long timestamp) {
> the existing one is the one that takes filtered data as a param, this function we are commenting on now is one that does not take filtered data and nothing actually ever calls this
> 
> —
> Reply to this email directly or view it on GitHub.
>